### DISCUSSION TITLE
Add modelOpen booleans to hooks

### DISCUSSION THREAD

**Discussion by lukevs** [rainbow-me/rainbowkit#614] [Ideas]
ConnectButton.Custom exposes:
- `accountModalOpen`
- `chainModalOpen`
- `connectModalOpen`

It would be nice to have these in the new modal hooks as well:
```
const { connectModalOpen, openConnectModal } = useConnectModal();
const { accountModalOpen, openAccountModal } = useAccountModal();
const { chainModelOpen, openChainModal } = useChainModal();
```

Quick PR here - https://github.com/lukevs/rainbowkit/pull/1

Curious if it's something others are up for!

**Comment by markdalgleish** [comment]
Are you making use of these booleans? I'm interested to know what they're being used for.

**Reply by malcodeman** [reply]
Hi,
In my case I am showing a loading spinner in the background when user opens the connect modal.
With `@web3modal/react` this was easy because they expose `isOpen` boolean but since I switched to rainbowkit I am forced to use `ConnectButton.Custom` which is fine but doing it with hooks would be cleaner.
